apps/monitoring/kube-prometheus-stack/dev/kustomization.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../kube-prometheus-stack/dev/prometheus-values.enc.yaml
```

The file is in the same folder, think you just want this right?
```suggestion
  - prometheus-values.enc.yaml
```

---

ah right yeah